src/miniapp.ts

```diff
@@ -0,0 +1,5 @@
+export declare namespace MiniApp {
```

Comment:
I see that we are exporting in 3 ways. Let's document it?

Reply:
Sure. Those exports are required to use it as a commonjs module.